mkdocs.yml

```diff
@@ -102,21 +103,30 @@ nav:
   - Resources:
     - Cloud Resources: Resources/Cloud_Resources/index.md
     - FPGA Resource: Resources/FPGA_Resources/index.md
-    - GPU Resources: Resources/GPU_Resources/index.md
+    - GPU Resources:
+      - Overview: Resources/GPU_Resources/index.md
+      - CMS Resources:
+        - SWAN: Resources/GPU_Resources/CMS_Resources/SWAN.md
+        - LXPLUS Condor: Resources/GPU_Resources/CMS_Resources/lxplus_condor.md
+        - LXPLUS GPU: Resources/GPU_Resources/CMS_Resources/lxplus_gpu.md
+        - ml.cern.ch: Resources/GPU_Resources/CMS_Resources/ml_cern_ch.md
+        - Notebooks:
```

Comment:
Why should these be on the sidebar rather than linked from a given document?

Reply:
When building the site with mkdocs, there are warnings that these pages exist but are not mentioned anywhere in the navbar in the YAML file. This fixes these warnings.
Also, from a user perspective, having them in the sidebar ensures one has to do fewer clicks to switch between these pages. Currently, once one decided to look at e.g. the lxplus GPU page, one would have to open various dropdowns to go to another resource, or go back with the browser.
My main idea here was to replicate the directory structure and always show what pages are available on a given level.

Reply:
There will most likely be more GPU resources in the future. I don't love the idea that this list will grow indefinitely and that will be dumped into the sidebar. I prefer having people go to the index page, which can be formatted appropriately, to see what is available. Having someone go back in their browser or use a second tab is not a huge burden.

As for the INFO lines (not WARNINGS), there is a whole discussion about this in https://github.com/mkdocs/mkdocs/issues/1755. I say we punt on doing anything about this until this is resolved at the MkDocs level.

Reply:
Alright, that's also fine, makes sense.